components/molecule/autosuggest/src/index.js

```diff
@@ -0,0 +1,271 @@
+/*
+ TODO
+ - check focus order
+ - arrows not working properly
+ - suggested list, remove items. default items, check integrity
+ - personal items on the list written by the user
+ - Get New Component for input tags and inlcude it on the solution
+ - Get New Component for label and include on the solution
+ - Custom icons
+ - responsive
+ - sass
+ - add new model as input
+ - cambiar a funciones stateles
+ - ie 11
+ */
+
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import OptionShown from './model'
+import MoleculeSelectList from './list/index'
+
+const BASE_CLASS = 'sui-MoleculeAutosuggest'
+
+const SIZES = {
+  SHORT: 'short',
+  LARGE: 'large'
+}
+
+const LIST_BEHAVIOURS = {
+  ON_FOCUS: 'focus',
+  ON_2_KEYS: '2_keys',
+  ON_3_KEYS: '3_keys',
+}
+
+// const TYPES_
+
+function isoString (str) {
+  return str.normalize('NFD').replace(/[\u0300-\u036f]/g, '').toLowerCase()
+}
+
+class MoleculeAutosuggest extends Component {
+  listSize = SIZES.SHORT
+  listBehaviour = LIST_BEHAVIOURS.ON_FOCUS
+  valueBeforeFocus = null
+  labelToOptionPosition = []
+  uniqueId = 'sui-MoleculeAutosuggest'
```

Comment:
This is not unique at all. It's the same for all instances.

Reply:
this var is reasigned on the constructor using `sui-MoleculeAutosuggest` as a base. I will refactor so is easier to understant